### ISSUE TITLE
Font awesome Library does not recognize modules and loads their showMissingIcons

### ISSUE THREAD

**Issue by bmartinez287** [UTCWeb/particle#160]
![Screen Shot 2021-01-26 at 3 41 16 PM](https://user-images.githubusercontent.com/39039024/105902701-fdce2c00-5fec-11eb-8fb4-e4e84b28de18.png)
![Screen Shot 2021-01-26 at 3 41 07 PM](https://user-images.githubusercontent.com/39039024/105902702-fe66c280-5fec-11eb-9236-1ba0f63bd560.png)

If javascript is on the scripts steps on those items.

**Comment by bmartinez287** [comment]
Particle has a way of importing fontawesome icons independently of the fontawesome module. The particle one is nice and useful for twig templates and css styles while the module leans towards user generated input.
We could use only the particle one but that could increase the built size plus it would make assumptions on how it should be used. Particle only uses 5 icons from fontawesome trough the fortawesome package and takes to tree shacking it loads only the 5 that are specified.

**Comment by bmartinez287** [comment]
https://fontawesome.com/how-to-use/javascript-api/setup/configuration
https://github.com/FortAwesome/react-fontawesome/issues/187

**Comment by bmartinez287** [comment]
I found out the answer is more involved that it look like at the beginning.
![Screen Shot 2021-01-28 at 3 58 30 PM](https://user-images.githubusercontent.com/39039024/106198642-4b2fd200-6182-11eb-915a-7c448632ef94.png)
Right now, there are 3 sources of font awesome that are competing with one another. The font-awesome showMissingIcons hide the console errors.
The issue is that all sources are justified and serve a different purpose.

1. The first source is inside themag under the thecharm subtheme. It basically styles items and subcategories.
2. The second source is the fontawesome module. Which adds a lot of cool ckeditor and drupal features.
3. The third source is particle, which uses web-pack to optimize its fontawesome .

The easiest way to resolve it, is to leverage the strengths each of them has.
The idea is to use the themag as the main source as is up in the hierarchy and has admin interface item. Then, patch the font-awesome icon so it won't import its libraries and use a different version. Finally, disable font-awesome inside particle.

I tested that workflow and we get no console errors and the library displays as expected so far.
